src/utils.jl

```diff
@@ -49,6 +67,36 @@ function pairwise_bhattacharyya(s1::Array{kmerdist},s2::Array{kmerdist})
 end
 
 
+function multi_pairwise_bhattacharyya(seq::Array{Array{Array{Float32}}}, sizes::Array{Int64})
+    costmat = zeros(sizes...)
+    searchpos = convert(Array{Int64},ones(length(seq)))
+    multi_fill_costmatrix(seq, costmat, searchpos)
+    return costmat
+end
+
+"""
+a function that recusivly fills out a cost matrix (costmat)
+"""
+function multi_fill_costmatrix(seq::Array{Array{Array{Float32}}}, costmat, searchpos)
+    #checks if thos point has not already been computed
```

Comment:
is this some sort of optimization for multiple sequence alignent? 

FIrst focus on getting it working in the normal - difficult case.